platform-api/che-core-api-runner/src/main/java/org/eclipse/che/api/runner/internal/Runner.java

```diff
@@ -90,6 +92,10 @@ public boolean isValid(DeploymentSources deployment) {
     private ScheduledExecutorService cleanScheduler;
     private java.io.File             deployDirectory;
 
+    @Inject
+    private DownloadPlugin theDownloadPlugin;
```

Comment:
@garagatyi the old field is final protected, not private. i saw it too risky to just remove it or hide non-incrementally.